File: Data Structures and Algorithms/Power.c
/*

#include<stdio.h>
int pow(int ,int );
int main()
{
	int m,n,res;
	printf("Enter the number : ");
	scanf("%d",&m);
	printf("Enter the Power for that Number : ");
	scanf("%d",&n);
	res=pow(m,n);
	printf("Power of %d to %d is %d",m,n,res);
	
}
int pow(int a,int b)
{
	if(b==0)
	return 1;
	else
	return a*pow(a,b-1);
}
*/
#include<stdio.h>
int pow(int ,int );
int main()
{
	int m,n,res;
	printf("Enter the number : ");
	scanf("%d",&m);
	printf("Enter the Power for that Number : ");
	scanf("%d",&n);
	res=pow(m,n);
	printf("Power of %d to %d is %d",m,n,res);
	
}
int pow(int a,int b)
{
	if(b==0)
	return 1;
	if(b%2==0)
	return pow(a*a,b/2);
	else
	return a*pow(a*a,(b-1)/2);
}

File: Java/Power.java
//WAP to find the power of a number without "Math.pow"
import java.io.*;
class Power
{ 
  public static void main(String[] args)throws IOException
  {
    int n,m,R=1,i;
	InputStreamReader ir=new InputStreamReader(System.in);
	BufferedReader br=new BufferedReader(ir);
	System.out.println("Enter the value of m,n");
	n=Integer.parseInt(br.readLine());
	m=Integer.parseInt(br.readLine());
	for(i=1;i<=n;i++)
	{
	R=R*m;
	}
	System.out.println("Result="+R);
	}
	}
	